README.md

```diff
@@ -78,6 +78,17 @@ If you want to position smart app banner yourself (e.g. in CSS), you can disable
 <meta name="smartbanner:disable-positioning" content="true">
 ```
 
+#### Define parent element
+
+Additionally to disabling positioning, you might want to place the smart app banner inside a specific container (e.g. parent element). The parent element is defined by a css selector:
+```html
+<meta name="smartbanner:disable-positioning" content="true">
+<meta name="smartbanner:parent-element" content="#mobile-navigation.banner-container">
```

Comment:
Unrelated to this project, but let's not promote overqualified selectors and go simpler:
```suggestion
<meta name="smartbanner:parent-element" content=".banner-container">
```